src/features/threads/AutoDraft.js

```diff
@@ -0,0 +1,139 @@
+const Feature = require("../../core/Feature");
```

Comment:
Use `Logger.debug` in place of these `console.logs` please.
They shouldn't pertain to the every day user, just debugging.